cli/pcluster/config/mappings.py

```diff
@@ -141,9 +141,11 @@
     "volume_types": ["standard", "io1", "gp2", "st1", "sc1"],
     "vpc_id": r"^vpc-[0-9a-z]{8}$|^vpc-[0-9a-z]{17}$",
     "deployment_type": ["SCRATCH_1", "SCRATCH_2", "PERSISTENT_1"],
-    "per_unit_storage_throughput": [50, 100, 200],
+    "per_unit_storage_throughput": [12, 40, 50, 100, 200],
```

Comment:
I'd define two keys:
```python
"fsx_ssd_throughput": [50, 100, 200],
"fsx_hdd_throughput": [12, 40],
```

In the `per_unit_storage_throughput` param you can use:
```python
"allowed_values": ALLOWED_VALUES["fsx_ssd_throughput"] + ALLOWED_VALUES["fsx_hdd_throughput"],
```

and then you can also reuse these values in the validators:
```python
if fsx_per_unit_storage_throughput not in ALLOWED_VALUES["fsx_hdd_throughput"]:
    errors.append("For HDD filesystems, per_unit_storage_throughput can only have the following values: {0}".format(ALLOWED_VALUES["fsx_hdd_throughput"]:))
```

This help us to have [dry](https://realpython.com/lessons/dry-principle/) code and have these numbers here and there in the code.

Reply:
Done with adding it in the mappings!
But for reusing these values in the validators, when I import ALLOWED_VALUE in mappings.py into validator.py, that cause a lots of unit tests in other test files failed.

Reply:
The problem is that in `mappings` file we're including the `validators`, so you cannot include the `mappings` in the `validators` one, it is a circulary dependency.

You can move the definition of these lists in the `constants.py` module, we already have, by defining `FSX_SSD_THROUGHPUT_ALLOWED_VALUES` and `FSX_HDD_THROUGHPUT_ALLOWED_VALUES` and reusing them in both the mappings and validators modules.

It is important to avoid to replicate this information because if in future there are changes we'll have a single place to update.

Reply:
Done!